pre_commit/commands/run.py

```diff
@@ -153,31 +153,12 @@ def _run_single_hook(
     filenames = classifier.filenames_for_hook(hook)
 
     if hook.id in skips or hook.alias in skips:
-        output.write(
-            _full_msg(
-                start=hook.name,
-                end_msg=SKIPPED,
```

Comment:
Is this `SKIPPED` variable used anywhere anymore? Can we yeet it too?

Reply:
Yes, but it's pretty yeetable. I should Not Be Lazy. Okay done.